src/lem/tag.rs

```diff
@@ -0,0 +1,45 @@
+use std::fmt::Display;
+
+use crate::field::LurkField;
+
+#[derive(Clone, Copy, PartialEq, Debug, PartialOrd, Ord, Eq, Hash)]
+pub enum Tag {
+    // expression tags
+    Dummy,
```

Comment:
This is worrisome.

Reply:
Okay, there's actually a `Dummy` tag, but it's a continuation tag